src/components/Reports/Reports.js

```diff
@@ -100,9 +102,16 @@ class Reports extends React.Component {
                 dateFrom={this.state.dateFrom}
                 isVisitsWithEmail={true}
               />
-            </>
-          ) : null}
-        </div>
+            </div>
+            <ReportsPageRanking
+              domainName={this.state.domainSelected.name}
+              dateTo={this.state.dateTo}
+              dateFrom={this.state.dateFrom}
+            />
+          </section>
+        ) : (
+          <div className="loading-box" />
```

Comment:
Could you use here the loading component?

Reply:
i talk with @GusBaamonde he told me use this div. this is another spinner

Reply:
The loading component is ready to render a different kind of spinners. Did you take a look?